Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1128 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.InterfacesLib;
+using Amdocs.Ginger.Repository;
+using GingerCore.Activities;
+using GingerCore.Variables;
+using QCRestClient;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using System.Web;
+using TDAPIOLELib;
+
+namespace GingerCore.ALM.Qtest
+{
+    public static class ImportFromQtest
+    {
+        static QTestApi.LoginApi connObj = new QTestApi.LoginApi();
+        static QTestApi.ProjectApi projectsApi = new QTestApi.ProjectApi();
+        static QTestApi.TestsuiteApi testsuiteApi = new QTestApi.TestsuiteApi();
+        static QTestApi.FieldApi fieldApi = new QTestApi.FieldApi();
+
+        static QTestApiClient.ApiClient apiClient = new QTestApiClient.ApiClient();
+        static QTestApiClient.Configuration configuration = new QTestApiClient.Configuration();
+
+        public static ObservableList<ActivitiesGroup> GingerActivitiesGroupsRepo { get; set; }
+        public static ObservableList<Activity> GingerActivitiesRepo { get; set; }
+        
+        public static QtestTestSuite ImportTestSuiteData(QtestTestSuite TS, long projectId)
+        {
+            testsuiteApi = new QTestApi.TestsuiteApi(connObj.Configuration);
+            QTestApiModel.TestSuiteWithCustomFieldResource testSuite = testsuiteApi.GetTestSuite(projectId, (long)Convert.ToInt32(TS.ID));
+            
+            // List TSTests = GetListTSTest(TS);
+
+            // foreach (TSTest tsTest in TSTests)
+            // {
+            //    TS.Tests.Add(ImportTSTest(tsTest)); //(equivalent to Activities)
+            // }
+
+            return TS;
+        }
+
+        public static QtestTest ImportTSTest(QtestTest tsTest)
+        {
+            QtestTest newTSTest = new QtestTest();
+            string linkedTest = string.Empty;
+
+            //Get the TC general details
+         
+            newTSTest.TestID = tsTest.TestID;
+            newTSTest.TestName = tsTest.TestName;
+
+            //Get the TC design steps
+            // List TSTestSteps = GetListTSTestSteps(tsTest);
+            foreach (QtestTestStep tsStep in tsTest.Steps)
+            {
+                QtestTestStep newtsStep = new QtestTestStep();
+                newtsStep.StepID = tsStep.StepID.ToString();
+                newtsStep.StepName = tsStep.StepName;
+                newtsStep.Description = tsStep.Description;
+                newtsStep.Expected = tsStep.Expected;
+                newTSTest.Steps.Add(newtsStep);
+            }
+
+            ////Get the TC parameters and their selected value
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)